Sources/Rendering/OpenGL/Texture/ComputeGradients.worker.js

```diff
@@ -61,12 +66,10 @@ registerWebworker(
 
           var mag = vec3.length(grad);
           vec3.normalize(grad, grad);
-          gradients[outPtr++] = grad[0];
-          gradients[outPtr++] = grad[1];
-          gradients[outPtr++] = grad[2];
-          gradients[outPtr++] = mag;
-          gradientMagnitudes[inPtr] = mag;
-          inPtr++;
+          gradients[outPtr++] = 127.5 + 127.5 * grad[0];
```

Comment:
A good rule of thumb for me is that when strange numbers like this appear in code, it's worthy of an inline comment explaining them.

Reply:
yep, I can add one

Reply:
Just say normal encoding, it is pretty standard in shaders